Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
```

Comment:
I'd find the omission surprising. Where is the line between dropping system dylibs and not dropping them from the output? And what's the bother with the extra output? All the binaries linking to it are in a separate indented part, so one can easily skip this entire block when scanning the output …